integration/spark/app/src/main/java/io/openlineage/spark/agent/OpenLineageSparkListener.java

```diff
@@ -269,14 +281,10 @@ public static void close() {
     clear();
   }
 
-  /**
-   * Check the {@link SparkConf} for open lineage configuration.
-   *
-   * @param applicationStart
-   */
   @Override
   public void onApplicationStart(SparkListenerApplicationStart applicationStart) {
     initializeContextFactoryIfNotInitialized(applicationStart.appName());
+    appStartCounter.increment();
```

Comment:
would it make sense to implement similar counter for other event types?

Reply:
Yes.